book/09-git-and-other-scms/sections/import-svn.asc

```diff
@@ -91,6 +91,14 @@ $ rm -Rf .git/refs/remotes
 ----
 
 Now all the old branches are real Git branches and all the old tags are real Git tags.
+
+Unfortunately `git svn` creates one extra branch named `trunk` for svn's trunk, which is essentially the same as the already created `master` branch (it actually refers to the same commit). Hence, let's remove this doublet:
```

Comment:
How about:

> There's one last thing to clean up. Unfortunately, `git svn` creates an extra branch named `trunk`, which maps to Subversion's default branch, but the `trunk` ref points to the same place as `master`. Since `master` is more idiomatically Git, here's how to remove the extra branch: